api/src/main/java/io/grpc/Grpc.java

```diff
@@ -62,4 +64,64 @@ private Grpc() {
   @Retention(RetentionPolicy.SOURCE)
   @Documented
   public @interface TransportAttr {}
+
+  /**
+   * Creates a channel with a target string and credentials. The target can be either a valid {@link
```

Comment:
=> Creates a channel *builder* ...

Reply:
This is actually the same language on ManagedChannelBuilder today :). Fixed.